src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/IConfiguredProjectHostObject.cs

```diff
@@ -11,5 +11,10 @@ internal interface IConfiguredProjectHostObject
         /// Display name for the configured project.
         /// </summary>
         string ProjectDisplayName { get; }
+
+        /// <summary>
+        /// Target framework moniker for the configured project.
+        /// </summary>
+        string TargetFrameworkMoniker { get; }
```

Comment:
You mean specifically this property? Nothing at the moment and I could get rid of it.